test/Microsoft.AspNetCore.Mvc.Core.Test/Internal/AttributeRouteTest.cs

```diff
@@ -230,35 +224,33 @@ public void AttributeRoute_GetEntries_CreatesLinkGenerationEntry_WithDefault()
                 },
             };
 
+            var builder = CreateBuilder();
             var actionDescriptorProvider = CreateActionDescriptorProvider(actions);
-
             var route = CreateRoute(CreateHandler().Object, actionDescriptorProvider.Object);
 
             // Act
-            var entries = route.GetEntries(actionDescriptorProvider.Object.ActionDescriptors);
+            route.AddEntries(builder, actionDescriptorProvider.Object.ActionDescriptors);
 
             // Assert
             Assert.Collection(
-                entries.LinkGenerationEntries,
+                builder.OutboundEntries,
                 e =>
                 {
-                    Assert.NotNull(e.Binder);
                     Assert.Empty(e.Constraints);
                     Assert.Equal(new RouteValueDictionary(new { slug = "hello" }), e.Defaults);
-                    Assert.Equal(RoutePrecedence.ComputeGenerated(e.Template), e.GenerationPrecedence);
-                    Assert.Equal("BLOG_INDEX", e.Name);
+                    Assert.Equal(RoutePrecedence.ComputeOutbound(e.RouteTemplate), e.Precedence);
```

Comment:
Unrelated to this change - shouldn't this verify the actual value (e.g. `Assert.Equal(4, e.Precedence);`) rather than verify it against the computed value?